packer/linux/conf/bin/bk-configure-docker.sh

```diff
@@ -66,7 +68,9 @@ elif [[ "${DOCKER_NETWORKING_PROTOCOL}" == "dualstack" ]]; then
   DOCKER_EXPERIMENTAL=true
   cat <<<"$(
     jq \
-      '.ipv6=true | ."fixed-cidr-v6"="2001:db8:1::/64" | .ip6tables=true | ."default-address-pools"=[{"base":"172.17.0.0/12","size":20},{"base":"192.168.0.0/16","size":24},{"base":"2001:db8:2::/104","size":112}]' \
+      --arg pool1 "${DOCKER_ADDRESS_POOL_1:-172.17.0.0/12}" \
```

Comment:
Should we make subnet configurable?

Reply:
This is acting as a fallback if the default params for `DockerAddressPool1` and `DockerAddressPool2` are emptied/removed.